modules/dynamic/test/src/smithy4s/dynamic/DummyIO.scala

```diff
@@ -0,0 +1,39 @@
+/*
+ *  Copyright 2021-2022 Disney Streaming
+ *
+ *  Licensed under the Tomorrow Open Source Technology License, Version 1.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *     https://disneystreaming.github.io/TOST-1.0.txt
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package smithy4s.dynamic
+import smithy4s.tests._
+
+object DummyIO {
```

Comment:
They call me Haxxor, destroyer of common-sense. 

Seriously though, it was easier to do that than turn everything into a `scala.util.Try`

Reply:
ahhaha that's clever